README.md

```diff
@@ -24,3 +24,35 @@ user.save!
 6. Fill the rest of the form and submit it.
 
 You're good to go!
+
+## Testing
+
+Run `rake decidim:generate_external_test_app` to generate a dummy application.
+
+Require missing factories in `spec/factories.rb`
+
+Add `require "rails_helper"` to your specs and execute them from the **root directory**, i.e.:
+
+`rspec decidim-process-extended/spec/serializers/decidim/participatory_processes/participatory_process_serializer_spec.rb`
+
+## Open Data
+
+Public information from Processes and Normative consultations can be accessed at [https://analisi.transparenciacatalunya.cat/](https://analisi.transparenciacatalunya.cat/dataset/Participa-Gencat/dazj-skq4), which is powered by [Socrata](https://socrata.com/).
```

Comment:
Regulations?

Reply:
I changed it to _Participatory processes_ to be more generic.

Now that I think of it, it should be documented in the main README that this app distinguishes between standard participatory processes and regulations, right?